microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/utils/JDTTypeUtils.java

```diff
@@ -105,6 +105,10 @@ public static boolean isList(String valueClass) {
 	public static boolean isNumber(String valueClass) {
 		return NUMBER_TYPES.contains(valueClass);
 	}
+	
+	public static boolean isBoolean(String valueClass) {
+		return valueClass.equals("boolean") || valueClass.equals("java.lang.Boolean");

```

Comment:
please remove java.lang.Boolean. Perhaps we should rename to isPrimitiveBoolean.

Reply:
sounds good